src/chromeServices/contentScript/site.ts

```diff
@@ -50,7 +51,8 @@ export class Site {
             validated: this.#siteConfig.validate?.call(this),
             newInputText: newInputText ?? this.#previousInput, //null check for newInputText
             inputTime: this.#inputTime,
-            source: this.#siteConfig.name
+            source: this.#siteConfig.name,
+            hidden: bool
```

Comment:
The `hidden` property could be added in the backend, rather than in the content script. `ITranslationSnapshot` contains all data scraped from the site, while `hidden` is an independently generated variable.